cirq-core/cirq/contrib/routing/initialization_test.py

```diff
@@ -39,6 +39,7 @@ def test_initialization_reproducible_with_seed(seed):
     eq.add_equality_group(*mappings)
 
 
+@pytest.mark.usefixtures('restore_random_state')
 @pytest.mark.parametrize('graph_seed,state', [(random.randint(0, 2**32), np.random.get_state())])
```

Comment:
> I don't understand why the np.random.get_state() call is here in the decorator.

Good point.  I will tweak it to get those values in the test body.

> Also, it's not clear why we have to call np.random.set_state internally at all...

I think the intent is to test `get_initial_mapping()` with `init_seed=None` -
https://github.com/quantumlib/Cirq/blob/791180c296055123be1d3b93e6258432872c8767/cirq-core/cirq/contrib/routing/initialization_test.py#L29
I did not dig into the code, but it presumably uses a global numpy RNG in such case.

The test then fails without the `np.random.set_state` call.  At any rate,
I can remove the `restore_random_state` fixture here, because there is no seeding of RNG to a constant.

Done in 4b89b70c2730ffb95504c7e21b194797f9ceca0e